modules/tile-converter/src/i3s-converter/i3s-converter.ts

```diff
@@ -1227,4 +1236,7 @@ export default class I3SConverter {
 
     this.refinementCounter.tilesCount += 1;
   }
+  private isContentSupported(sourceRootTile) {
```

Comment:
Add a JSDoc comment. The name is a bit unclear. Supported how? As in "has images"?

I assume the class in this file handles all types of conversion including tile types that this function considers "not supported".